docs/tutorial.md

```diff
@@ -0,0 +1,93 @@
+
+# CLN Tutorial
+
+This tutorial will guide you through the issuance, interaction, and exchange of community coins (CC). To follow this tutorial you should have a basic understanding of Ethereum and blockhain technologies. You can start from the [Ethereum wiki](https://github.com/ethereum/wiki/wiki/Ethereum-introduction) or this [awesome list](https://github.com/ethereum/wiki/wiki/Ethereum-introduction) of links currated by the community. This [overview](overview.md) gives a high level explanation of the CLN conracts, and a [contract reference](reference) provides a self generated documentation similar to REST API docs.
+
+From a practical side you will need an ERC20 compatible wallet to send and sign transactions. I will use [MyEtherWallet](https://www.myetherwallet.com/) because of his abillity to generate transactions from ABI. Also of course you will need ETH to pay for gas, and CLN to interact with contracts of Colu Local Network. You can learn more about the relation of CLN and Community Currenies (CC) from the [whitepapper](https://cln.network/pdf/cln_whitepaper.pdf).
+
+To proceed with this guide I created a new Ethereum account - [0x0d4DF041Dbef6fFC0E444a4a213774AdB0c118C2](https://ropsten.etherscan.io/address/0x0d4DF041Dbef6fFC0E444a4a213774AdB0c118C2). I'm going to perform all the action on Ropsten Test Network, so you can follow. We encourage you first to use testnet untill a sufficient understanding of the CLN contracts is achieved.
+
+## Contract Addresses
+
+#### Ropsten Test Network
+- [ColuLocalNetwork](reference/ColuLocalNetwork.md) -  [0x41c9d91e96b933b74ae21bcbb617369cbe022530](https://ropsten.etherscan.io/address/0x41c9d91e96b933b74ae21bcbb617369cbe022530)
```

Comment:
add ABIs files and links to them

Reply:
They should get ABI from the etherscan, I instruct how to do this in the manual